eq-author/src/App/page/Logic/Routing/RoutingPage/NoRouting.js

```diff
@@ -33,34 +36,47 @@ const Paragraph = styled.p`
 
 const AddRoutingButton = styled(Button)`
   margin: 2em auto 1em;
+  ${props =>
+    props.isLastPage &&
+    `
+     pointer-events: none;
+     opacity: 0.6;`};
 `;
 
-const RoutingRuleSetMsg = ({
-  title,
-  onAddRouting,
-  children,
-  ...otherProps
-}) => (
-  <Container {...otherProps}>
-    <Icon />
-    <Title>{title}</Title>
-    <Paragraph>{children}</Paragraph>
-    <AddRoutingButton
-      small
-      naked
-      variant="primary"
-      onClick={onAddRouting}
-      data-test="btn-add-routing"
-    >
-      <IconText icon={IconAddRule}>Add your first rule</IconText>
-    </AddRoutingButton>
-  </Container>
-);
+const RoutingRuleSetMsg = ({ onAddRouting, page, ...otherProps }) => {
+  let lastSection, lastSectionPage, lastPageId;
+  if (otherProps.questionnaire) {
+    const questionnaire = otherProps.questionnaire;
+    lastSection = questionnaire.sections.length;
+    lastSectionPage = questionnaire.sections[lastSection - 1].pages.length;
+    lastPageId = questionnaire.sections[lastSection - 1].pages[lastSectionPage - 1].id;
+  };
+  const currentPageId = page && page.id;
```

Comment:
Doing `&&` here will create a boolean instead of returning the page ID. If this is intended, I'd suggest renaming the variable to make it clearer that it's not storing the actual ID.

```suggestion
  const currentPageId = page.id;
```

If you want to check that `page` isn't null or undefined before you try and do `page.id`, you could do this instead:

`const currentPageId = page ? page.id : null;` 

or

`const currentPageId = page.id || null;`

You might even want to throw an error if `page` isn't there - although (I might be wrong about this next bit) we assume that it would be, as the front end will always be sending it.

```
if(!page){
    throw new error("Page cannot be undefined or null");
}

const currentPageId = page.id;
```